src/plugins/exportFile/types/_base.js

```diff
@@ -46,7 +46,8 @@ class BaseType {
   /**
    * Merge options provided by users with defaults.
    *
-   * @return {Object} Returns new options object.
+   * @returns {object} Returns new options object.
```

Comment:
Swap places @returns and @param, please.

Reply:
Resolved within https://github.com/handsontable/handsontable/commit/902cb8ba8578dcc38dc0d342bd222a703c32a68a.